### DISCUSSION TITLE
Get a real fraction when you are writing 3/4

### DISCUSSION THREAD

**Discussion by ghost** [KaTeX/KaTeX#3727] [Q&A]
I want to be able to write real fractions and other math symbols. For instance when I write 3/4 it becomes a real fraction and when I write 5^3 it shows as 5 powered by 3.

Is that possible to be done? Has someone ever accomplished this feature or something related?

**Comment by ronkok** [comment]
Two suggestions:

1. [AsciiMath](http://asciimath.org/)
2. My own effort in this field is at [Hurmet.app](https://hurmet.app/examples.html)

**Reply by ghost** [reply]
Hey, your work seems great...
Do you have any React.js wrapper component to use with your rich text editor?

**Reply by ronkok** [reply]
Nope.

**Comment by mbourne** [comment]
Further to Ron Kok's comment (Hurmet's great!), you may be  interested to see my IntMath where I use AsciiMath throughout. I take the LaTeX output that AsciiMath produces  and feed it to KaTeX.

Here's a typical page: https://www.intmath.com/fourier-series/3-fourier-even-odd-functions.php

You may also be interested in this page https://www.intmath.com/help/asciimath-input-latex-katex-output.php where the user can enter simple AsciiMath expressions and see both LaTeX and kaTeX output. The page source will show how it's done.

Hope it's useful.

Regards
Murray
